tests/test_moneytracker/test_models.py

```diff
@@ -34,65 +76,163 @@ def test_customuser_not_suscribed_creation(self, mock_save):
     """
-    Test that a PortfolioAsset object was instantiated successfully using mocks
+    Test that a PortfolioAsset object was instantiated.
+
+    successfully using mocks.
     """
-    @patch('moneytracker.models.PortfolioAsset.save')
-    @patch('moneytracker.models.PortfolioAsset.portfolio')
-    @patch('moneytracker.models.PortfolioAsset.asset')
-    def test_portfolioasset_creation(self, mock_asset, mock_portfolio, mock_save):
+
+    @patch("moneytracker.models.PortfolioAsset.save")
+    @patch("moneytracker.models.PortfolioAsset.portfolio")
+    @patch("moneytracker.models.PortfolioAsset.asset")
+    def test_portfolioasset_creation(
+        self,
+        mock_asset,
+        mock_portfolio,
+        mock_save
+    ):
+        """
+        Test the creation of a PortfolioAsset.
+
+        Parameters:
+            self (TestClass): The instance of the test class.
+            mock_asset (MagicMock): The mocked asset object.
+            mock_portfolio (MagicMock): The mocked portfolio object.
+            mock_save (MagicMock): The mocked save method.
+
+        Returns:
+            None
+        """
         mock_asset.name = "Mocked Asset"
         mock_portfolio.user.__str__.return_value = "Mocked Portfolio"
-        portfolioasset_example = PortfolioAsset(portfolio=mock_portfolio, asset=mock_asset, quantity=1)
+        portfolioasset_example = PortfolioAsset(
+            portfolio=mock_portfolio, asset=mock_asset, quantity=1
+        )
 
         self.assertEqual(portfolioasset_example.portfolio, mock_portfolio)
         self.assertEqual(portfolioasset_example.asset, mock_asset)
         self.assertEqual(portfolioasset_example.quantity, 1)
-        self.assertEqual(str(portfolioasset_example), 'Mocked Asset portfolio of Mocked Portfolio')
+        self.assertEqual(
+            str(portfolioasset_example),
+            "Mocked Asset portfolio of Mocked Portfolio"
+        )
         mock_save.assert_not_called()
 
 
 class PerformanceModelTest(TestCase):
     """
-    Test that a Performance object was instantiated successfully using mocks
+    Test that a Performance object was instantiated.
+
+    successfully using mocks.
     """
-    @patch('moneytracker.models.Performance.save')
-    @patch('moneytracker.models.Performance.user')
+
+    @patch("moneytracker.models.Performance.save")
+    @patch("moneytracker.models.Performance.user")
     def test_performance_creation(self, mock_user, mock_save):
+        """
+        Test the creation of a Performance object.
+
+        This test function checks if a Performance object is created
+        correctly by verifying that:
+        - The user attribute of the Performance object is set correctly.
+        - The days_to_send_email attribute of the Performance object
+            is set correctly.
+        - The __str__ method of the user object returns the expected value.
+        - The save method of the Performance object is not called.
```

Comment:
I love this format of comments, please could you edit the anothers for the same 'list description'